src/GLM.jl

```diff
@@ -1,12 +1,13 @@
 __precompile__()
 
 module GLM
+    using ArgCheck
```

Comment:
The uses of ArgCheck here are so simple that I don't think it's worth adding a dependency for a couple of cases. (As an aside, if we were to add a dependency it would need to go in the REQUIRE file.)